backend/server.py

```diff
@@ -34,10 +39,33 @@ def get_tree_by_id(id):
             internalSrvErr.description = str(e)
             raise internalSrvErr
 
+    @app.route("/get-node-info/<node_id>", methods=["GET"])
+    def get_node_info(node_id):
+        """
+        Gets the associated link from the database, gets the text associated
+        with it, and summarizes it.
+        """
+
+        def _get_content_from_site(url):
+            page = requests.get(url)
+            soup = BeautifulSoup(page.content, "html.parser")
+            content = soup.find_all(["h1", "h2", "p"])
+            return "\n".join([c.text for c in content])
```

Comment:
Is there any better way to format this? In the frontend I can't see any paragraphs and the article is difficult to read due to this

Reply:
[UPCOMING TICKET]